cmd/entrypoint/main.go

```diff
@@ -39,9 +40,39 @@ var (
 	waitPollingInterval = time.Second
 )
 
+func cp(src, dst string) error {
+	s, err := os.Open(src)
+	if err != nil {
+		return err
+	}
+	defer s.Close()
+
+	d, err := os.OpenFile(dst, os.O_RDWR|os.O_CREATE, 0777)
```

Comment:
is 777 what we want? isnt that 644 the default?

Reply:
The default for [`os.Create`](https://golang.org/pkg/os/#Create) is 0666. With that, or 0644, subsequent steps fail with `exec: \"/tekton/tools/entrypoint\": permission denied": unknown` because the other group doesn't have permission to execute.

0667 works, or as little as 0311 (owner can write, anybody can execute), but I didn't really feel strongly about limiting permissions on the executable. I can knock it down to 0311 though.